dcs/task.py

```diff
@@ -162,8 +162,8 @@ def stop_after_duration(self, duration: int):
         self.params["stopCondition"]["duration"] = duration
 
 
-class WeaponType(Enum):
-    Auto = 1073741822
+class WeaponType(IntEnum):
+    Auto = 9663676414
```

Comment:
This is sort of alarming, but I created a test miz ([weapon type.zip](https://github.com/pydcs/dcs/files/13942649/weapon.type.zip)) and it agrees. I wonder what DCS has been doing with the wrong value...

Reply:
I'd be able to merge that now rather than whenever the carpet bombing task is ready.

Reply:
@DanAlbert - sumitted as a new PR.